packages/xod-project/src/messages.js

```diff
@@ -188,6 +260,10 @@ export default {
     note: 'The program has a cycle',
     solution: 'Use xod/core/defer node to break the cycle',
   }),
+  BAD_LITERAL_VALUE: ({ value }) => ({
+    title: 'Bad literal value',
+    note: `${value} is not a valid literal`,
```

Comment:
Solution is:

`If you meant a string, surround it with double quotes: "${literal}".`

Check, however `packages/xod-project/src/messages.js` → `INVALID_LITERAL_BOUND_TO_PIN`. Looks like code duplication.

Reply:
`BAD_LITERAL_VALUE` is returned from `getTypeFromLiteral` which takes just a `DataValue` and does not know about pins etc.